test/test.js

```diff
@@ -92,6 +92,14 @@ var BufferList = require("bl");
           if (entry.fileName !== expectedName) {
             throw new Error("unexpected entry fileName: " + entry.fileName + ", expected: " + expectedName);
           }
+          var mode = entry.externalFileAttributes >>> 16;
+          if (/\/$/.test(entry.fileName)) {
```

Comment:
```suggestion
          if (entry.fileName.startsWith('/')) {
```

Reply:
The other spot in this file is using that regex to decide if it's a directory, so just matching that style, though I agree it's not particularly readable.  So much so that your suggestion is actually backwards (should be `fileName.endsWith('/')`) ^_^.  `endsWith`, however, is ES6+ only, and doesn't exist in Node v0.10, that this library is still being tested against.